src/components/SearchResults.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "../styles/searchResults.css";
+import PropTypes from "prop-types";
+
+// eslint-disable-next-line react/function-component-definition
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p className="para-fail">No results</p>;
+    // eslint-disable-next-line no-else-return
+  } else {
+    return (
+      <>
+        {results.map((image) => (
+          <>
+            <div className="all-images" />
+            <img className="space-image" src={image} alt="mock" key={image} />
+          </>
+        ))}
+      </>
```

Comment:
This is a bit confusing... Here you are returning a fragment that contains another fragment, which in turn contains a self closing div (which never has anything inside it), which is supposed to be the container for the images, then below that the image in the map. This whole thing is repeated for every image in the props passed in. I think what you were going for was something like this:
```
    <div className="all-images">
      {results.map((image) => {
        return (
          <img className="space-image" src={image} alt="mock" key={image} />
        );
      })}
    </div>
```
This will add a single container div (with className `all-images`) and render the images inside it

Also, the `mock` tag is pretty useless as all the images will have the same tag! Ideally this info should be coming in from the API, but this would be more of a nice to have feature here to be added when the other issues are fixed